system-parachains/bridge-hubs/bridge-hub-kusama/primitives/src/lib.rs

```diff
@@ -87,15 +87,15 @@ frame_support::parameter_types! {
 	/// The XCM fee that is paid for executing XCM program (with `ExportMessage` instruction) at the Kusama
 	/// BridgeHub.
 	/// (initially was calculated by test `BridgeHubKusama::can_calculate_weight_for_paid_export_message_with_reserve_transfer` + `33%`)
-	pub const BridgeHubKusamaBaseXcmFeeInKsms: u128 = 16_196_533_317;
+	pub const BridgeHubKusamaBaseXcmFeeInKsms: u128 = 16_156_041_984;
```

Comment:
Because [here](https://github.com/polkadot-fellows/runtimes/pull/159/commits/a0a61ae1e9613b6e9a6079dd9ad0ecbb341e89e3) I added improved sanity checks for these fees.

No, no, we won't change it on every release, only if those tests fail, which would mean that the fees can be actually lowered.